### ISSUE TITLE
Anagrafica Produttori

### ISSUE THREAD

**Issue by claudiogiare** [Jazzo/iGruppi#24]
Prevederei fin da subito di inserire dei campi aggiuntivi nella maschera dati produttore per abbozzare una SCHEDA PRODUTTORE, molto utile per la presentazione del produttore agli altri GAS (es. GAS Nuovi)
I campi potrebbero essere, per adesso,  tutti dei "semplici" campi descrittivi
Nome Produttore
Contatti  (ci sono già dettagliati, ok)
Riferimento per i GAS  (persona che tiene i contatti con i GAS)
Sito Web
Presentazione produttore
Tipologia Prodotto
Dettaglio prodotti (descrizione)
Certificazioni 
Produzione 
Modalità di Distribuzione per i GAS 
Collaborazione con i GAS
Possibilità di incontrare il produttore
Indicazioni per il Periodo ordine
Listini
Indicazioni per i Prezzi riservati ai GAS
Note
Links utili per conoscere il produttore

SONO TUTTI CAMPI FACOLTATIVI, ma stimolano il referente a conoscere meglio il produttore e a condividere queste informazioni importanti che saranno facilmente a disposizione di tutti (gasisti dello stesso GAS ed altri GAS).
In futuro potremo implementare mappa, foto, Logo, e creare una vera e propria scheda produttore per i GAS scaricabile in PDF.

**Comment by claudiogiare** [comment]
Inseriamo anche un campo 
Data Ultimo aggiornamento anagrafica:

**Comment by Jazzo** [comment]
Ok, però non starei a crearli tutti come singoli campi della tabella, mi
sembra eccessivo.
Possiamo gestire buona parte di questi dati in un *campo note esteso e
formattato HTML* con titoli, sottotitoli, ecc.
Terrei fuori sitoweb e certificazioni (se gestibile con voci fisse in menù
a tendina altrimenti anche lui nel campo note).

2015-03-24 11:12 GMT+01:00 claudiogiare <notifications@github.com>:

> Inseriamo anche un campo
> Data Ultimo aggiornamento anagrafica:
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Jazzo/iGruppi/issues/24#issuecomment-85434404>.
>



-- 
Davide Gullo, Consulente Web (professionista disciplinato ai sensi della
legge 4/2013)
http://www.m4ss.net
gullo@m4ss.net

**Comment by claudiogiare** [comment]
La distinzione dei campi era voluta solo per una maggiore chiarezza e per dare una traccia al referente produttore ed invitarlo a trovare/compilare tutte queste informazioni che sono importanti per un GAS ed anche per il suo ruolo ("conoscere meglio il Produttore). 
Il campo NOTE unico e generico rischia di limitare l'inserimento delle informazioni, con conseguente moltiplicazioni di mail per domande al referente (spesso le stesse).

La gestione di questi dati in un *campo note esteso e formattato HTML* con titoli, sottotitoli, ecc. può essere una buona soluzione (anche dal punto di vista grafico), ma bisognerebbe trovare il modo di presentare comunque la "traccia" al referente produttore che lo compila.

Potrebbe essere bello raggruppare queste info all'interno di una specifico sezione TAB (o cmq al posto del tab Note) con scritto "SCHEDA PRODUTTORE" con tutti questi campi/informazioni a disposizione.
Sarebbe uno spazio a disposizione del referente produttore (in verità del Gestore, ma perchè no anche di tutti i referenti produttore dei vari gruppi) per aggiornare di volta in volta le informazioni, anche a seguito delle domande che vengono dagli altri gasisti.
Così, per ogni gasista, si trasmetterà l'abitudine che quando devo fare una domanda che riguarda un produttore, prima vado a vedere la scheda e solo se non trovo risposte, scrivo al referente produttore.
Comunque non c'è fretta e ne riparliamo.

**Comment by StefanoMenozzi** [comment]
Concordo anche io sui campi. Capisco Davide che in prima fase finché non sono chiari i campi, preferisce mettere un macro campo note. Però la scheda divisa permetterà di fare tanti ragionamenti.
Io introdurrei anche ... la butto lì, ma andrà forse ampliato molto come concetto... quello di un algoritmo di feedback e di rating del produttore o addirittura del prodotto... con criteri di visibilità da pensare bene per evitare di offendere qualche produttore.

**Comment by StefanoMenozzi** [comment]
Campo: tipi di pagamento accordati al gruppo
